apps/meteor/server/services/omnichannel/queue.ts

```diff
@@ -45,6 +51,7 @@ export class OmnichannelQueue implements IOmnichannelQueue {
 		await LivechatInquiry.unlockAll();
 
 		this.running = false;
+		this.lock = Promise.resolve();
```

Comment:
this will just remove the reference to the existing stack of calls to `start`, without preventing them from running. 
If a new call is made afterwards, it'll not wait for the existing stack of calls to complete.